components/suggest/src/store.rs

```diff
@@ -48,6 +56,25 @@ use crate::{
 pub struct SuggestStore {
     inner: SuggestStoreInner<remote_settings::Client>,
 }
+#[derive(Deserialize, Serialize, Default, Debug)]
+#[serde(transparent)]
+pub struct UnparsableRecords(BTreeMap<String, UnparsableRecord>);
+
+impl FromSql for UnparsableRecords {
+    fn column_result(value: rusqlite::types::ValueRef<'_>) -> rusqlite::types::FromSqlResult<Self> {
+        Ok(serde_json::from_str(value.as_str()?).unwrap())
```

Comment:
I might use a different error type here:

```suggestion
        serde_json::from_str(value.as_str()?)
            .map_err(|err| rusqlite::types::FromSqlError::Other(Box::new(err)))
```

...But yes, it's a great suggestion to avoid `unwrap` here! 😊